user-interface-document/index.html

```diff
@@ -4,97 +4,150 @@
+							<i>Flavorful Tuscany Meetup</i>. We hope you will
+							enjoy the program as well as your stay at the
+							<a href="http://ckeditor.com">Bilancino Hotel</a>.
+						</p>
+						<figure class="image">
+							<img
+								src="https://ckeditor.com/assets/images/ckdemo/editor-types/malta.jpg"
+								srcset="
+									https://ckeditor.com/assets/images/ckdemo/editor-types/malta.jpg,
+									https://ckeditor.com/assets/images/ckdemo/editor-types/malta_2x.jpg 2x
+								"
+								alt="Bilancino Hotel"
+							/>
+							<figcaption>Bilancino Hotel</figcaption>
+						</figure>
+						<p>Please find below the full schedule of the event.</p>
 
-					<figure class="table">
-						<table>
-							<thead>
-								<tr>
-									<th colspan="2">Saturday, July 14</th>
-								</tr>
-							</thead>
-							<tbody>
-								<tr>
-									<td>9:30 AM - 11:30 AM</td>
-									<td>
-										<p><strong>Americano vs. Brewed</strong> - “know your coffee” with:&nbsp;</p>
-										<ul style="list-style-type:circle;">
-											<li>Giulia Orlanda</li>
-											<li>Stefano Garau</li>
-											<li>Giuseppe Russo</li>
-										</ul>
-									</td>
-								</tr>
-								<tr>
-									<td>1:00 PM - 3:00 PM</td>
-									<td>
-										<p><strong>Regional delicacies of Tuscany</strong> - live cooking</p>
-										<p>Incorporate the freshest ingredients&nbsp;<br>with Rita Fresco</p>
-									</td>
-								</tr>
-								<tr>
-									<td>5:00 PM - 8:00 PM</td>
-									<td><strong>Tuscan vineyards at a glance</strong> - wine-tasting&nbsp;<br>with Frederico Riscoli</td>
-								</tr>
-							</tbody>
-						</table>
-					</figure>
+						<figure class="table">
+							<table>
+								<thead>
+									<tr>
+										<th colspan="2">Saturday, July 14</th>
+									</tr>
+								</thead>
+								<tbody>
+									<tr>
+										<td>9:30 AM - 11:30 AM</td>
+										<td>
+											<p>
+												<strong
+													>Americano vs.
+													Brewed</strong
+												>
+												- “know your coffee” with:&nbsp;
+											</p>
+											<ul style="list-style-type: circle">
+												<li>Giulia Orlanda</li>
+												<li>Stefano Garau</li>
+												<li>Giuseppe Russo</li>
+											</ul>
+										</td>
+									</tr>
+									<tr>
+										<td>1:00 PM - 3:00 PM</td>
+										<td>
+											<p>
+												<strong
+													>Regional delicacies of
+													Tuscany</strong
+												>
+												- live cooking
+											</p>
+											<p>
+												Incorporate the freshest
+												ingredients&nbsp;<br />with Rita
+												Fresco
+											</p>
+										</td>
+									</tr>
+									<tr>
+										<td>5:00 PM - 8:00 PM</td>
+										<td>
+											<strong
+												>Tuscan vineyards at a
+												glance</strong
+											>
+											- wine-tasting&nbsp;<br />with
+											Frederico Riscoli
```

Comment:
Is there _anything_ we could change in configuration to make the line splitting look a bit better? If anyone takes a look at it, it seems broken at the first glance 😞

Reply:
done